subgraphs/_reference_/schema.graphql

```diff
@@ -0,0 +1,557 @@
+enum Network {
```

Comment:
I suggest adding a line here showing which exact schema template does it extend.

In this case, something like "Extend schema-dex-amm.graphql, see the diff at the bottom"